mitosheet/src/types.tsx

```diff
@@ -541,6 +542,8 @@ export type FormatTypeObj =
  * @param renderCount - the number of times this sheet has rendered. Note that this is per instance of the backend object, 
  *        so it increments by one when the page the sheet is rendered on is refreshed, but resets to zero if a new
  *        mitosheet.sheet() call is made (even if it replays the analysis), as this is a new backend object.
+ * @param lastResult - This is the result of the last step that was applied. This might be undefined if the 
+ *        step does not return a result
```

Comment:
If I just used the drop duplicates taskpane so that X rows were removed, and then I edit a cell value, an event that does not return a result, the button in the drop duplicates switches to "Removed 0 rows". Can we store that result in the taskpane maybe so this doesn't happen?

Reply:
The editing taskpanes are not being closed properly. I'll fix this.